widgets/gui/qt_live_control.py

```diff
@@ -45,37 +49,51 @@ def __init__(self, parent, button_name):
         print("Multithreading with maximum %d threads" % self.q_thread_pool.maxThreadCount())
 
     def launch_nidaq_instance(self):
-        if self.state_tracker:
-            self.trigger_stop_live.emit()  # does nothing on first iteration before thread is made.
-            # Stops thread before new one is launched. Needed when instanced on parameter change.
-            # Not needed in timelapse
 
-            parameters = self.parent.left_window.update_parameters
-            view = self.view_combobox.currentText()
-            channel = self.laser_combobox.currentText()
+        self.trigger_stop_live.emit()  # does nothing on first iteration before thread is made.
+        # Stops thread before new one is launched. Needed when instanced on parameter change.
+        # Not needed in timelapse
 
-            print("called with:", parameters, view, "and channel",
-                  channel)
+        print("back at wait")
 
-            # launch worker thread with newest parameters
-            daq_card_thread = qt_nidaq_worker.NIDaqWorker(parameters, view, channel)
-            # connect
-            self.trigger_stop_live.connect(daq_card_thread.stop)
+        while True:
+            print("waiting", self.wait_shutdown)
+            time.sleep(1)
+            # time.sleep(2)
+            if not self.wait_shutdown:
+                break
+            QApplication.processEvents()
```

Comment:
this is the line that does the trick, check my changes @ljjh20  and let me know if there are any unclear part